.betterer.results

```diff
@@ -3446,6 +3446,12 @@ exports[`better eslint`] = {
     "public/app/features/dashboard/dashgrid/DashboardPanel.tsx:5381": [
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"]
     ],
+    "public/app/features/dashboard/dashgrid/PanelPerformanceMonitor.tsx:5381": [
+      [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "1"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "2"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "3"]
+    ],
```

Comment:
Do you plan on fixing these soon so we can remove betterer?

Reply:
Just pushed a commit that fixes all betterer results. Not sure if it's the best way to fix, would appreciate your input. Thanks!

Reply:
nice stuff! chef kiss